Makefile

```diff
@@ -85,6 +85,19 @@ ifeq ($(INSTALLATION_TYPE), managed-api)
 	INSTALLATION_SHORTHAND ?= rhoam
 endif
 
+ifeq ($(INSTALLATION_TYPE), multitenant-managed-api)
```

Comment:
I wonder if we should define this in separate files, and name them after the install type, and can we just include them then? 
e.g.
`multitenant-managed-api.env`:
```
PROJECT=managed-api-service
TAG ?= $(RHOAM_TAG)
OPERATOR_IMAGE=$(REG)/$(ORG)/$(PROJECT):v$(TAG)
NAMESPACE_PREFIX ?= redhat-rhoam-
APPLICATION_REPO ?= managed-api-service
INSTALLATION_PREFIX ?= redhat-rhoam
OLM_TYPE ?= multitenant-managed-api-service
INSTALLATION_NAME ?= rhoam
INSTALLATION_SHORTHAND ?= rhoam
```
and replace the below with:
```
include $(INSTALLATION_TYPE).env
```
WDYT?

Reply:
I really like this idea, updated it now.